### PR TITLE
Fix eserver

### PR DESCRIPTION
**Pull request by giggsoff**
Fixes for uploading files into eserver: sha256 on fly, check for file existence.

Argument `--image-size` for `eden setup` to create large images of EVE

### CODE REVIEW

File: cmd/downloader.go [hunk 1]
```diff
@@ -51,7 +51,7 @@ var downloadEVECmd = &cobra.Command{
 		if devModel == defaults.DefaultGCPModel {
 			format = "gcp"
 		}
-		if err := utils.DownloadEveLive(adamDist, eveImageFile, eveArch, eveHV, eveTag, eveUefiTag, format); err != nil {
+		if err := utils.DownloadEveLive(adamDist, eveImageFile, eveArch, eveHV, eveTag, eveUefiTag, format, eveImageSizeMB); err != nil {
```
Comment (mydatascience) [inline cmd/downloader.go:54]: Can we find out it ourselves?

**Review by mydatascience — COMMENTED** [review]
File size is definitely not better (I'd say worse) than sha

File: cmd/downloader.go (hunk 1 above)
Comment (giggsoff) [reply cmd/downloader.go:54]: We need to do it on step of setup. Cannot catch your idea, please elaborate.

**Comment by giggsoff** [comment]
>File size is definitely not better (I'd say worse) than sha

Yes, it is. We calculate sha in eserver. In this PR calculation done in the same time with uploading.

**Review by rvs — APPROVED** [review]
LGTM

**Pull request merged**